config/routes.rb

```diff
@@ -4,6 +4,10 @@
   devise_for :admins, path: 'admins'
   devise_for :employees, path: 'employees'
   resources :kudos
-    get '/admin' => "pages#dashboard", :as => :admin_root
+  namespace :admins do
+    resources :kudos
+  end
+  resources :pages
```

Comment:
what do you think about moving `:pages` to admins namespace, as this is the main page for the admin dashboard?

also, we should limit the endpoint to only index resources if we only use the index in the controller. 

we shouldn't leave any unused endpoints in routes